pom.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.github.camel-tooling</groupId>
+	<artifactId>camel-tooling-common</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
```

Comment:
don't we want to align somehow with server and clients?

Reply:
I think that we don't care.
Using 0.0.1 version is a better practice I think that starting with with 1.0.0. it will let us more freedom to change API.